WorldMoodTracker/js/components/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from '../../../css/bootstrap.min.css';
+import custom from '../../css/style.css';
+
+export default class Footer extends React.Component {
+    render() {
+        return (
+            <footer>
+                <div className={styles.container}>
+                    <div className={[styles.hiddenLg, styles.hiddenMd, styles.hiddenSm, styles.textCenter,
+                        custom.footerLink].join(' ')}>
+                        <a href="/">Home</a>
```

Comment:
I don't get it, why is home taking me to your git repo? I think **Home** is not an appropriate word there.